src/app/components/TransactionsTable/index.tsx

```diff
@@ -49,12 +52,13 @@ export default function TransactionsTable({ transactions }: Props) {
                         {tx.title}
```

Comment:
I think when title is blank then the type and date below move up. 
maybe we should always use a fixed height here? (always add a &nbsp; at the end 😱 )

Reply:
Fixed

Reply:
lol, did not know that GH interprets `&nbsp;` here in comments.

Reply:
<img width="454" alt="image" src="https://user-images.githubusercontent.com/318/180186890-e832b67a-5854-458e-b2cd-c7464150648b.png">